src/generate/apiCollection.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ApiGroup } from "./apiGroup";
+
+/**
+ * A collection of multiple groups of APIs stored by key.
+ */
+export class ApiCollection extends Map {
```

Comment:
You said that you were going to change back to not extending Map. Did you change your mind?

Reply:
Ha, so far, yes. They're still iterable in Handlebars which I think is enough.